configs/peer/genesis.json

```diff
@@ -70,6 +70,9 @@
                   "PermissionTokenDefinition": {
                     "id": {
                       "name": "can_unregister_asset_with_definition"
+                    },
+                    "params": {
+                      "asset_definition_id": "Id"
```

Comment:
These changes need to be  communicated to all of our downstream users.

Reply:
Of course.

As soon as this PR will be merged I'll notify our SDK developers and docs writers.